pkgs/development/libraries/ortp/default.nix

```diff
@@ -6,15 +6,15 @@
 
 stdenv.mkDerivation rec {
   pname = "ortp";
-  version = "4.5.1";
+  version = "5.0.66";
```

Comment:
5.1.3 is the latest one as for now.

Reply:
5.1.12 actually (though that's [the same commit](https://gitlab.linphone.org/BC/public/ortp/-/commit/23439fba3b800c2fdb105af748ca018f51146d57)). I have made [that change in my update to 4.4.1](https://github.com/ThibaultLemaire/nixpkgs/commit/e57e1465d90f03082a52e38a73ce477aa1fb0636#diff-a0c8d04f47ef57ec3f17101235177d5624be63fd87d4124dc4ebbff874f0f8f8R10).

However linphone 4.3.2 has ortp pinned to [13731952b44192eb9be3e517a370df05c449dd28](https://gitlab.linphone.org/BC/public/ortp/-/commit/13731952b44192eb9be3e517a370df05c449dd28) so 5.0.66 is adequate here.

Reply:
I see, thanks.